createst.py

```diff
@@ -421,6 +423,38 @@ def eve2test():
         return
     filter_event_type_params(eve_rules=content)
 
+    output_path = os.path.join(test_dir, "test.yaml")
+
+    with open(output_path, "r") as f:
```

Comment:
As the function documentation states, this function is doing something completely unrelated to parsing or processing the command-line arguments provided to the createst script. This doesn't look like the right place to add these changes.

And indeed, if your patch is run, the resulting output shows that ;)